src/main/java/teammates/common/datatransfer/FeedbackQuestionType.java

```diff
@@ -104,12 +106,79 @@ public FeedbackResponseDetails getFeedbackResponseDetailsInstance(FeedbackQuesti
         try {
             feedbackResponseDetails.extractResponseDetails(this, questionDetails, answer);
         } catch (Exception e) {
+            e.printStackTrace();
             Utils.getLogger().warning("Failed to extract response details.\n" + e.toString());
             return null;
         }
 
         return feedbackResponseDetails;
     }
+    
+    /**
+     * Returns an instance of a corresponding Feedback*ResponseDetails class
+     *
+     * @return FeedbackResponseDetails
+     */
+    public FeedbackResponseDetails getFeedbackResponseDetailsInstance(FeedbackQuestionDetails questionDetails,
```

Comment:
This method overloads getFeedbackResponseDetailsInstance, but it seems like the original getFeedbackResponseDetailsInstance is not used anymore? If that's the case, then we can remove it, and also update the tests for that to test this instead.